main.py

```diff
@@ -119,6 +59,8 @@ def ParseTimestamps(self, lines):  #returns timestamps for lines
                     dtt = time.strptime(match.group(1), '%b %d %Y %H:%M:%S UTC')
                 elif type == 2:
                     dtt = time.strptime(match.group(1), '[%m/%d/%Y %I:%M:%S %p]')
+                elif type == 3:
+                    dtt = time.strptime(match.group(1), '[%Y-%m-%d %I:%M:%S UTC]')
```

Comment:
Will your service eventually normalize the timestamps so i don't have to handle all the cases here?